builds/misc/templates/build-api-proxy.yaml

```diff
@@ -0,0 +1,24 @@
+parameters:
+  name: 'API proxy'
```

Comment:
nit: Since these parameters are not passing in, suggest to use variable instead of parameter if possible.